lib/main.dart

```diff
@@ -1,16 +1,19 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:flutter_svg/svg.dart';
 import 'package:flutter_toolbox/flutter_toolbox.dart';
 import 'package:seeds/screens/app/app.dart';
+import 'package:seeds/screens/onboarding/helpers.dart';
```

Comment:
Imports aren't in alfabet sort

Reply:
Oh - forgive us - we will fix it in next release